client/homebrew/editor/editor.jsx

```diff
@@ -299,7 +299,10 @@ const Editor = createClass({
 					style={{ display: 'none' }}
 					rerenderParent={this.rerenderParent} />
 				<MetadataEditor
-					metadata={this.props.brew}
+					metadata={({
+						...this.props.brew,
+						tags : typeof this.props.brew.tags === 'string' ? [] : this.props.brew.tags
+					})}
```

Comment:
I wonder if we can have this type checking handled upon receiving the brew to begin with. Make sure we have the `brew` in a good format right from the start instead of deep in here where it's kind of hidden. Especially where we might want to access `tags` from elsewhere and don't want to have to do this type check every time.

Reply:
So handle it on the backend when it sets req.brew?

Reply:
Done!